### PR TITLE
Awesome books classes

### PR DESCRIPTION
**Pull request by Theodoraldo**
At this stage, we
- Added classes to book functionality.
- Update CSS file.
- Update index.html file.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -3,46 +3,58 @@ const author = document.querySelector('.author');
...
+      title: this.btitle,
+      author: this.bauthor,
+    };
+    dataStore.push(Books);
+    updateData();
+    fetchData();
+  }
 }
 
 // Removing  data from localStorage
```
Comment (Omar-Muhamad) [inline main.js:60]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md#:~:text=it%20inside%20classes.-,Create%20class%20methods%20to%20add%20and%20remove%20books.,-Maintain%20all%20of), Kindly add this `removeItem` function as a method to the `NewBook` class you created above

File: main.js [hunk 2]
```diff
@@ -3,46 +3,58 @@ const author = document.querySelector('.author');
...
-      <hr />
-    `;
+      <div class="displayRow">
+        <ul>
+          <li>${items[i].title} by ${items[i].author}</li>
+        </ul>
+        <button onclick='removeitem(${i})'>Remove</button>
+      </div>
+      <hr/>
+      `;
```
Comment (Omar-Muhamad) [inline main.js:30]: - [ ] [Optional] You can also add this function as a method inside `NewBook` class you created.

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Theodoraldo and @AJ193,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Reem-lab — APPROVED** [review]
# Hi @Theodoraldo and @AJ193,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯


## Highlights 🔥🔥🔥🔥
- Followed GitHub flow ✅
- Great working on the functionality 🔥🔥
- Great working on the requested changes 🚀





## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**